packages/site-admin/package.json

```diff
@@ -0,0 +1,66 @@
+{
+	"name": "@automattic/site-admin",
+	"version": "0.0.1-beta.2",
+	"description": "A modern UI framework for building WordPress administrative interfaces.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"types": "dist/types/index.d.ts",
+	"calypso:src": "src/index.ts",
+	"sideEffects": false,
+	"scripts": {
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets",
+		"prepack": "yarn run clean && yarn run build",
+		"watch": "tsc --build ./tsconfig.json --watch"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"keywords": [
+		"admin",
+		"ui",
+		"components",
+		"framework",
+		"dashboard",
+		"react",
+		"wp-admin"
+	],
+	"publishConfig": {
+		"access": "public"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/site-admin"
+	},
+	"author": "Automattic Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": "https://github.com/Automattic/wp-calypso/issues",
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/site-admin#readme",
+	"peerDependencies": {
+		"react": "^18.3.1",
+		"react-dom": "^18.3.1"
+	},
+	"dependencies": {
+		"@wordpress/base-styles": "^5.17.0",
+		"@wordpress/commands": "^1.17.0",
+		"@wordpress/components": "^29.3.0",
+		"@wordpress/data": "^10.17.0",
+		"@wordpress/dom": "^4.17.0",
+		"@wordpress/edit-site": "^6.17.0",
```

Comment:
Why do we depend on edit-site? That is not a package that should be a dependency of any other package.

Reply:
I use the package to get the site settings. I didn't know we could not use it, mainly because the getSettings() is not private.

Reply:
> This package is meant to be used only with WordPress core. Feel free to use it in your own project but please keep in mind that it might never get fully documented.

til. we need to work on a different solution